src/SmartComponents/Inventory/applications/vulnerabilities/vulnerabilities.scss

```diff
@@ -28,9 +28,16 @@
 #download-report {
```

Comment:
Try to avoid styling using ID tags due to specificity levels

Reply:
It's not the (my) code related to this pull request. Please create a new issue.